### PR TITLE
Proximal game results

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #827

Extends work of @danflapjax from that PR to make it ready to merge, assuming others agree.

### CODE REVIEW

**Comment by prabau** [comment]
P76: Can you add some more specific places in the paper where the topological version of the game is defined?  Is it Game 1.11 or 1.12 or ... ?  And the equivalence with Bell's version is Theorem 1.13, I suppose.

And also in the `refs:` add the author's name in parentheses.
And how about separate the first sentence into its own paragraph, to make things easier to read?

**Comment by prabau** [comment]
Will take a look by the end of the day.

**Comment by prabau** [comment]
Question for my own benefit, as I am not as familiar with uniformities as I'd like to be.
Is the "universal uniformity" the same as what Willard calls the "fine uniformity"?

**Comment by StevenClontz** [comment]
> Question for my own benefit, as I am not as familiar with uniformities as I'd like to be. Is the "universal uniformity" the same as what Willard calls the "fine uniformity"?

Yes. It is the union of all compatible uniformities, itself a compatible uniformity.

**Comment by prabau** [comment]
The description of the game in P76 seems to be for Game 1.11 instead of Game 1.12 ?  Game 1.12 uses the $E_n$ which are the intersection of the $D_n$ played so far.

Is my understanding below correct?
I guess the Tactic paper means the description in Game 1.11/1.12 is "purely topological" because no choice of uniformity is involved, i.e., given a Tychonoff space, one has to work with the corresponding universal uniformity.

Bell's definition of proximal is for a uniform space instead, i.e. $(X,\mathscr U)$ with a fixed uniformity.  And the Tactic paper rephrases the definition for topological spaces, saying a top. space is proximal is there exists *some* compatible uniformity that is proximal. 

And then it proves that the topological space $X$ is proximal in that sense iff the corresponding universal uniformity is proximal in Bell's sense.   (Game 1.11 is the same as Game 1.2 with the universal uniformity for the space, right?)

So is it possible that a topological space $X$ is proximal, but some (non-universal) compatible uniformity is not proximal?
I.e., only some of the compatible uniformities are proximal (although the universal one always is).

File: properties/P000076.md [hunk 1]
```diff
@@ -2,9 +2,27 @@
...
-See {{doi:10.1016/j.topol.2014.05.010}} for a purely topological
-definition.
+
+A {P12} space such that there exists a uniformity
+generating the topology of the space such that the
+entourage-picker has a winning strategy in Bell's proximal game introduced
+in {{doi:10.1016/j.topol.2014.06.014}}.
+
+Equivalently, the entourage-picker has a winning strategy for
+this variation defined as Game 1.12 of
```
Comment (StevenClontz) [inline properties/P000076.md:17]:
```suggestion
this variation defined as Game 1.11 of
```

**Comment by StevenClontz** [comment]
@prabau you've got it. 

Jocelyn and I actually hit this recently, where we found a situation where we showed a certain kind of topological group was not proximal with respect to its left or right uniformities, but that didn't rule out that it was proximal with respect to the universal uniformity, ie as a topological space. On the other hand, compact spaces only have one compatible uniformity, so we did have the result in that scenario.

**Comment by prabau** [comment]
Cool.  That's all for my comments.  I'll let @danflapjax do the review.

**Review by danflapjax — APPROVED** [review]
The description for proximal is much better; great improvement :+1:

**Pull request merged**